js/src/forum/components/DiscussionPage.js

```diff
@@ -96,34 +94,6 @@ export default class DiscussionPage extends Page {
     );
   }
 
-  onbeforeupdate(vnode) {
```

Comment:
Looks like this implicitly includes #2377? Not opposed to it, but since that's going away the mentions PR will still be needed here.

Reply:
Yes, `m.route.set` would trigger a full re-render, so it needs to be replaced with the stream `goTo` functions.